edu.princeton.physics.WSJTX.yml

```diff
@@ -0,0 +1,50 @@
+app-id: edu.princeton.physics.WSJTX
+runtime: org.kde.Platform
+runtime-version: '5.15'
+sdk: org.kde.Sdk
+command: wsjtx
+rename-desktop-file: wsjtx.desktop
+rename-icon: wsjtx_icon
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --device=all
+  - --socket=pulseaudio
+  - --filesystem=home
+  - --share=network
+cleanup:
+  - /man
+  - /include
+  - /share/man
+  - /share/doc
+  - /doc
+  - '*.a'
+  - '*.la'
+modules:
+  - shared-modules/libusb/libusb.json
+  - hamlib.yml
```

Comment:
it's not common practice to simply split modules like that.

Reply:
Yeah, flat structure for short-to-medium sized manifests is preferred.

Reply:
Inlined